integrations/flow-judge.md

```diff
@@ -0,0 +1,125 @@
+version: Haystack 2.0
+toc: true
+---
+### **Table of Contents**
+- [Overview](#overview)
+- [Installation](#installation)
+- [Usage](#usage)
+- [License](#license)
+
+## Overview
+This integration allows you to evaluate Haystack pipelines using Flow Judge.
+
+Flow Judge is an open-source, lightweight (3.8B) language model optimized for LLM system evaluations. Crafted for accuracy, speed, and customization.
+
+Read the technical report [here](https://www.flow-ai.com/blog/flow-judge).
+
+## Installation
+
+For running Flow Judge with vLLM engine:
+```bash
+pip install flow-judge[vllm]
+pip install 'flash_attn>=2.6.3' --no-build-isolation
+```
+For running Flow Judge with transformers:
+```bash
+pip install flow-judge[hf]
+```
+If flash attention:
+```bash
+pip install 'flash_attn>=2.6.3' --no-build-isolation
+```
+For running Flow Judge with Llamafile on macOS:
+```bash
+pip install flow-judge[llamafile]
+pip install 'flash_attn>=2.6.3' --no-build-isolation
+```
+
+Finally install Haystack:
+```bash
+pip install haystack-ai
+```
+
+## Usage 
+Flow Judge integration with Haystack is designed to facilitate the evaluation of Haystack pipelines using Flow Judge. This integration allows you to seamlessly integrate Flow Judge into your Haystack workflows, enabling you to evaluate and improve your LLM systems with precision and efficiency. 
+
+Flow Judge offers a set-of built-in metrics and easy-to-create custom metrics. 
+
+### Components
+This integration introduces `HaystackFlowJudge` component.
+  
+### Use Flow Judge with Haystack 
+We have created a comprehensive guide on how to effectively use Flow Judge with Haystack. You can access it [here](https://github.com/flowaicom/flow-judge/blob/main/examples/5_evaluate_haystack_rag_pipeline.ipynb). This tutorial demonstrates how to evaluate a RAG pipeline built with Haystack using Flow Judge. 
+
+### Quick Example
+The code snippet below provides a simpler example of how to integrate Flow Judge with Haystack. However, we recommend following the full tutorial for a deeper understanding of the concepts and implementation. 
+
+```python
+from flow_judge.integrations.haystack import HaystackFlowJudge
+from flow_judge.metrics.presets import RESPONSE_FAITHFULNESS_5POINT
+from flow_judge import Hf
+
+from haystack import Pipeline
+
+# Create a model using Hugging Face Transformers with Flash Attention
+model = Hf() # We support also Vllm, Llamafile
+
+# Evaluation sample 
+questions = ["What is the termination clause in the contract?"] 
+contexts = ["This contract may be terminated by either party upon providing thirty (30) days written notice to the other party. In the event of a breach of contract, the non-breaching party may terminate the contract immediately."]
+answers = ["The contract can be terminated by either party with thirty days written notice."] 
+
+# Define the HaystackFlowJudge evaluator, we will use the built-in metric for faithfulness
+ff_evaluator = HaystackFlowJudge(
```

Comment:
Any api reference/docs page documenting the parameters of `HaystackFlowJudge` component?